js_modules/dagster-ui/packages/ui-core/src/testing/mocking.ts

```diff
@@ -0,0 +1,145 @@
+import {DocumentNode} from '@apollo/client';
```

Comment:
This is a carbon copy of the module from cloud, except the __typenames are `Query` instead of `CloudQuery`